<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VlansOnInterface</class>
 <widget class="QMainWindow" name="VlansOnInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="PolicyList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>131</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReloadBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CommitBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Commit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
